static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -37,7 +37,7 @@ define([
                         askQuestionBtn.classList.add('is-hidden');
                     }
 
-                    if (isEmailSubmissionReady === true) {
+                    if (isEmailSubmissionReady === "true") {
```

Comment:
Uggh... So I assume that `isEmailSubmissionReadyElement.dataset.isEmailSubmissionReady` is a string of `"true"`? 🤕